integrationTests/src/test/java/org/sonarqube/gradle/GradleTest.java

```diff
@@ -258,7 +258,8 @@ public void testProjectWithConfigurationCacheDoubleExecutionsShouldWork() throws
   @Test
   public void testKotlinMultiplatformProject() throws Exception {
     Assume.assumeTrue("Tests only applies to version 6.8.3 or greater", getGradleVersion().isGreaterThanOrEqualTo("6.8.3"));
-    Properties props = runGradlewSonarSimulationModeWithEnv("/kotlin-multiplatform", emptyMap(), "compileCommonMainKotlinMetadata", "compileKotlinJvm", "compileKotlinMetadata", "compileTestKotlinJvm");
+    Properties props = runGradlewSonarSimulationModeWithEnv("/kotlin-multiplatform", emptyMap(), "compileCommonMainKotlinMetadata", "compileKotlinJvm", "compileKotlinMetadata",
+      "compileTestKotlinJvm");
```

Comment:
Let's revert this change
```suggestion
    Properties props = runGradlewSonarSimulationModeWithEnv("/kotlin-multiplatform", emptyMap(), "compileCommonMainKotlinMetadata", "compileKotlinJvm", "compileKotlinMetadata", "compileTestKotlinJvm");
```